### PR TITLE
📝 Add Documentation About Dashboard

### PR DESCRIPTION
**Pull request by Paulomart**
## What did you change?

Add documentation about the dashboard view.

## How can others test the changes?

Read

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [ ] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: bpmn-studio/components/dashboard/dashboard.md [hunk 1]
```diff
@@ -1,8 +1,33 @@
 # Dashboard
 
+## Aufgaben und Motivation
+
+Ziel des Dashboards ist es den Benutzer einen schnellen Überblick über die
```
Comment (robpal94) [inline bpmn-studio/components/dashboard/dashboard.md]: _**dem** Benutzer_

File: bpmn-studio/components/dashboard/dashboard.md [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
+aktuell laufenden Prozesse zu geben. Diese Informationen werden von der aktuell
+verbunden ProcessEngine bezogen.
+
+## Aufbau und Strukturierung
+
+Das Dashboard kann über den Navigationseintrag "Dashboard" angesteuert werden.
+
+In dem Dashboard gibt folgende zwei Abschnitte:
```
Comment (robpal94) [inline bpmn-studio/components/dashboard/dashboard.md]: _Im Dashboard gibt es die beiden folgenden Abschnitte_

File: bpmn-studio/components/dashboard/dashboard.md [hunk 3]
```diff
@@ -1,8 +1,33 @@
...
+
 ![Dashboard](dashboard.png)
 
-Das Dashboard ist in der Navigationsleiste zu finden und zeigt die aktuell
-laufenden Prozesse der verbundenen ProcessEngine. Zusätzlich wird eine Liste
-der anstehenden UserTasks angezeigt. Es gibt die Möglichkeit UserTasks einer
-bestimmten Korrelation anzuzeigen.
+## Funktion
+
+In der Tabelle "Processes running" wird für jede laufende Prozessinstanze zwei
```
Comment (robpal94) [inline bpmn-studio/components/dashboard/dashboard.md]: _Prozessinstanz_

File: bpmn-studio/components/dashboard/dashboard.md [hunk 4]
```diff
@@ -1,8 +1,33 @@
...
+Links angeboten. In der Spalte `Process Model Id` ist jeweils ein Link zu der
+Design-Ansicht des Prozessmodels verlinkt. Ebenfalls können alle UserTasks für
+einen aktiven Prozess angezeigt werden.
+
+Die "Task waiting" Tabelle enthält für jede wartende UserTask einen `Continue`
+Button. Dieser dient dazu diese Task fortzuführen.
```
Comment (robpal94) [inline bpmn-studio/components/dashboard/dashboard.md]: _die entsprechende Task fortzuführen_.

File: bpmn-studio/components/dashboard/dashboard.md [hunk 5]
```diff
@@ -1,8 +1,33 @@
...
+
+Das Dashboard kann über den Navigationseintrag `Dashboard` geöffnet werden.
+
+Das Dashboard besteht aus folgenden zwei Abschnitten:
+
+Im ersten Abschnitt, `Processes running`, werden die laufenden Prozesse als
+Tabelle aufgelistet. In der Tabelle wird die `Correlation Id` und die
+`Process Model Id` angezeigt.
+
+Im zweiten Abschnitt, `Task waiting`, werden UserTasks angezeigt, die auf
```
Comment (alexanderkasten) [inline bpmn-studio/components/dashboard/dashboard.md]: Tasks

File: bpmn-studio/components/dashboard/dashboard.md [hunk 6]
```diff
@@ -1,8 +1,33 @@
...
+
 ![Dashboard](dashboard.png)
 
-Das Dashboard ist in der Navigationsleiste zu finden und zeigt die aktuell
-laufenden Prozesse der verbundenen ProcessEngine. Zusätzlich wird eine Liste
-der anstehenden UserTasks angezeigt. Es gibt die Möglichkeit UserTasks einer
-bestimmten Korrelation anzuzeigen.
+## Funktion
+
+In der Tabelle `Processes running` wird für jede laufende Prozessinstanz zwei
```
Comment (alexanderkasten) [inline bpmn-studio/components/dashboard/dashboard.md]: Prozessinstanz soll vermieden werden <= @5ebastianMeier

Comment (alexanderkasten) [reply bpmn-studio/components/dashboard/dashboard.md]: *werden

File: bpmn-studio/components/dashboard/dashboard.md [hunk 7]
```diff
@@ -1,8 +1,33 @@
...
+Links angeboten. In der Spalte `Process Model Id` ist jeweils ein Link zu der
+Design-Ansicht des Prozessmodels verlinkt. Ebenfalls können alle UserTasks für
+einen aktiven Prozess angezeigt werden.
+
+Die `Task waiting` Tabelle enthält für jede wartende UserTask einen `Continue`
+Button. Dieser dient dazu die entsprechende Task fortzuführen.
```
Comment (alexanderkasten) [inline bpmn-studio/components/dashboard/dashboard.md]: statt Task lieber Aufgabe verwenden.

File: bpmn-studio/components/dashboard/dashboard.md [hunk 8]
```diff
@@ -1,8 +1,33 @@
...
 
-Das Dashboard ist in der Navigationsleiste zu finden und zeigt die aktuell
-laufenden Prozesse der verbundenen ProcessEngine. Zusätzlich wird eine Liste
-der anstehenden UserTasks angezeigt. Es gibt die Möglichkeit UserTasks einer
-bestimmten Korrelation anzuzeigen.
+## Funktion
+
+In der Tabelle `Processes running` wird für jede laufende Prozessinstanz zwei
+Links angeboten. In der Spalte `Process Model Id` ist jeweils ein Link zu der
+Design-Ansicht des Prozessmodels verlinkt. Ebenfalls können alle UserTasks für
```
Comment (alexanderkasten) [inline bpmn-studio/components/dashboard/dashboard.md]: Hier wäre vielleicht noch gut zu wissen, wie ich die UserTasks anzeige, da hier nicht beschrieben ist, dass diese auch über einen entsprechenden Link zu erreichen sind.

**Pull request closed without merging**